viewer/src/main/webapp/WEB-INF/jsp/app.jsp

```diff
@@ -99,8 +99,23 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                 </c:otherwise>
             </c:choose>
         </c:if>
-        <script type="text/javascript" src="${contextPath}/viewer-html/common/overrides.js"></script>
-        <script type="text/javascript" src="${contextPath}/resources/js/FileSaver.js"></script>
```

Comment:
Why are these removed? and if these are no longer necessary - which I doubt- why aren't these files deleted?